browser_history/browsers.py

```diff
@@ -133,6 +133,22 @@ def bookmarks_parser(self, bookmark_path):
         return date_bookmarks
 
 
+class LibreWolf(Firefox):
+    """LibreWolf Browser
+
+    Supported platforms:
+
+    * Linux
+
+
+    Profile support: Yes
+    """
+    name = "LibreWolf"
+    aliases = ("librewolfurl",)
+
+    linux_path = ".librewolf/9ikbnwr6.default"
```

Comment:
The `9ikbnwr6.default` in the path is your profile's name. It won't be the same on all systems. For example, on my system it is `cx7612s2.default`.

Just `.librewolf` will work here. It will look for profiles inside there.

```suggestion
    linux_path = ".librewolf
```

Reply:
I also thought of it. I am adding a new pathset.